var/spack/repos/builtin/packages/geopm-service/package.py

```diff
@@ -0,0 +1,152 @@
+
+    Note: GEOPM interfaces with hardware using Model Specific Registers (MSRs).
+    For proper usage make sure MSRs are made available via the msr or
+    msr-safe kernel modules by your administrator."""
+
+    homepage = "https://geopm.github.io"
+    git = "https://github.com/geopm/geopm.git"
+    url="https://github.com/geopm/geopm/tarball/v3.0.1"
+
+    maintainers("bgeltz", "cmcantalupo")
+    license("BSD-3-Clause")
+    tags = ["e4s"]
+
+    version("develop", branch="dev", get_full_repo=True)
+    version(
+        "3.0.1",
+        sha256="32ba1948de58815ee055470dcdea64593d1113a6cad70ce00ab0286c127f8234",
+    )
+
+    variant("debug", default=False, description="Enable debug")
+    variant("docs", default=True, description="Create man pages with Sphinx")
+    variant(
+        "systemd",
+        default=False,
+        description="Enable use of systemd (systemd development libraries required)",
+    )
+    variant("liburing", default=True, description="Enables the use of liburing for batch I/O")
+    variant(
+        "libcap", default=True, description="Enables the use of libcap to do capabilities checks"
+    )
+    variant("gnu-ld", default=False, description="Assume C compiler uses gnu-ld")
+
+    variant("levelzero", default=False, description="Enables the use of oneAPI Level Zero loader")
+    variant("nvml", default=False, description="Enable NVML support")
+
+    variant(
+        "rawmsr",
+        default=True,
+        description="Enable direct use of standard msr device driver",
+        when="@develop",
+    )
+
+    conflicts("+nvml", when="+levelzero", msg="LevelZero and NVML support are mutually exclusive")
+
+    conflicts("%gcc@:7.2", msg="Requires C++17 support")
+    conflicts("%clang@:4", msg="Requires C++17 support")
+
+    conflicts("platform=darwin", msg="Darwin is not supported")
+    conflicts("platform=windows", msg="Windows is not supported")
+
+    conflicts("target=aarch64:", msg="Only available on x86_64")
+    conflicts("target=ppc64:", msg="Only available on x86_64")
+    conflicts("target=ppc64le:", msg="Only available on x86_64")
+
+    patch("0001-Support-NVML-via-CUDA-installation.patch", when="+nvml")
+
+    # Autotools dependencies
+    depends_on("automake", type="build")
+    depends_on("libtool", type="build")
+    depends_on("file")
+
+    # Docs dependencies
+    depends_on("doxygen", type="build", when="+docs")
+    depends_on("py-docstring-parser@0.13.0:", type="build", when="+docs")
+    depends_on("py-sphinx@4.5:", type="build", when="+docs")
+    depends_on("py-sphinx-rtd-theme@1:", type="build", when="+docs")
+    depends_on("py-sphinx-emoji@0.2.0:", type="build", when="+docs")
+    depends_on("py-sphinx-tabs@3.3.1:", type="build", when="+docs")
+    depends_on("py-pygments@2.13.0:", type="build", when="+docs")
+
+    # Other Python dependencies - from service/setup.py
+    depends_on("python@3.6:3", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("python@3.6:", type=("build", "run"))
```
Could just remove this line, Spack only supports Python 3.8+ anyway